index.html

```diff
@@ -12,123 +12,125 @@
-                hurdles, return .</p>
-            <p><code>
-                function hurdleRace(k, height) {
-                    let maxHeight = Math.max(...height);
-                    return (maxHeight > k)? maxHeight-k: 0;
-                }
-            </code></p>
-            <li>More... </li>
+            </section>
+            <section class="main-section" id="The_hurdle_race">
+                <header>
+                    <h2>The hurdle race</h2>
+                </header>
+                <p>A video player plays a game in which the character competes in a hurdle race. Hurdles are of varying
+                    heights, and the characters have a maximum height they can jump. There is a magic potion they can
+                    take
+                    that will increase their maximum jump height by unit for each dose. How many doses of the potion
+                    must
+                    the character take to be able to jump all of the hurdles. If the character can already clear all of
+                    the
+                    hurdles, return .</p>
+                <p><code>
+                        function hurdleRace(k, height) {
+                        let maxHeight = Math.max(...height);
+                        return (maxHeight > k)? maxHeight-k: 0;
+                        }
+                    </code></p>
 
-        </section>
-        <section class="main-section" id="Simple_Array_Sum">
-            <header>Simple Array Sum</header>
-            <p>Given an array of integers, find the sum of its elements.
-                For example, if the array 
-                <code>ar = [1,2,3], 1 + 2 + 3 = 6</code>
-            </p>
-            <p>Complete the simpleArraySum function in the editor below. 
-                It must return the sum of the array elements as an integer.
-                <br>simpleArraySum has the following parameter(s): </p>
+            </section>
+            <section class="main-section" id="Simple_Array_Sum">
+                <header>
+                    <h2>Simple Array Sum</h2>
+                </header>
+                <p>Given an array of integers, find the sum of its elements.
+                    For example, if the array
+                    <code>ar = [1,2,3], 1 + 2 + 3 = 6</code>
+                </p>
+                <p>Complete the simpleArraySum function in the editor below.
+                    It must return the sum of the array elements as an integer.
+                    <br>simpleArraySum has the following parameter(s):
+                </p>
                 <br>
-                <li> ar: an array of integers </li>
                 <code>function simpleArraySum(ar) {
-    
-                return ar.reduce((a, b) => a + b);}
-                    
+
+                    return ar.reduce((a, b) => a + b);}
+
                 </code>
 
-        </section>
-    </div>
+            </section>
+        </div>
     </main>
```

Comment:
Please, to improve readability separate logical blocks of code with new lines.
You can read good practices here [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).